docs/python/tutorial-azure-functions.md

```diff
@@ -0,0 +1,348 @@
+
+    ![Create new project button in the Functions explorer](images/functions/project-create-new.png)
+
+1. In the prompts that follow:
+
+    - Specify the folder for the project (the default is the current folder open in VS Code)
+    - Select **Python** for the language.
+    - Select **HTTP trigger** for the template. A function that uses an HTTP trigger is run whenever there's an HTTP request made to the function's endpoint. (You can see that there are a variety of other triggers for Azure Functions. To learn more, see [What can I do with Functions?](https://docs.microsoft.com/azure/azure-functions/functions-overview#what-can-i-do-with-functions) in the Azure documentation.)
+    - Name your function "HttpExample" (rather than accepting the default "HTTPTrigger") to distinguish the function itself from the trigger. This name is used for a subfolder that contains the function's code along with configuration data, and also defines the name of the HTTP endpoint.
+    - Select **Anonymous** for the authorization level, which makes the function publicly accessible to anyone.
+
+1. After a short time you see a message that the new project was created. In the **Explorer** you see the subfolder created for the function, and VS Code opens the `__init__.py` file that contains the default function code:
+
+    ![Result of creating a new Python functions project](images/functions/project-create-results.png)
+
+1. When `__init__.py` opens, VS Code might tell you that you don't have a Python interpreter selected. In that case, use the **Python: Select Interpreter** command from the Command Palette and select the virtual environment in the local `.env` folder (which was created as part of the project).
+
+    ![Selecting the virtual environment created with the project](images/functions/select-venv-interpreter.png)
+
+> **Tip**: Whenever you want to create another function in the same project, use the **Create Function** command in the **Azure: Functions** explorer, or use the **Azure Functions: Create Function** command from the Command Palette. Both commands prompt you for a function name (which is the name of the endpoint), then creates a subfolder with the default files.
+>
+> ![New Function command in the Azure: Functions explorer](images/functions/function-create-new.png)
+
+## Examine the code files
+
+In the newly-created function subfolder you see three files: `__init__.py`, which contains the function's code, `functions.json`, which describes the function to Azure Functions, and `sample.dat`, which demonstrates that a function subfolder can contain additional files if needs be. You won't be using `sample.dat`, so you can delete it if you want.
+
+Let's look at `functions.json` first, then the code in `__init__.py`.
+
+### functions.json
+
+The functions.json file provides the necessary configuration information for the function itself:
```

Comment:
In general make sure that there's some kind of way to distinguish between `Function` (Azure Function) and `function` (Python function object). This seems to be a problem we encounter a lot when writing Functions documentation, maybe check with Glenn for style advice?

Reply:
There are far too many occurrences of "function" that means "Azure Function" to use the proper name without being overburdensome. Instead, there are only a handful of places where I can distinguish Python functions specifically.